src/osd/ECUtil.h

```diff
@@ -32,6 +32,8 @@ const uint64_t CHUNK_INFO = 8;
 const uint64_t CHUNK_PADDING = 8;
 const uint64_t CHUNK_OVERHEAD = 16; // INFO + PADDING
 
+const std::string HINFO_KEY = "hinfo_key";
```

Comment:
but the declaration is not supposed to be visible to other compilation units, iiuc. a smaller and self-contained header file would be helpful to reduce the compiling speed, and in general to minimize cross dependencies, imho.